src/Jackett.Common/Indexers/AniLibria.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using AngleSharp.Html.Parser;
+using AngleSharp.Html.Dom;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Jackett.Common.Utils.Clients;
+using Newtonsoft.Json.Linq;
+using NLog;
+using System.Text.RegularExpressions;
+
+namespace Jackett.Common.Indexers
+{
+    [ExcludeFromCodeCoverage]
+    public class AniLibria : BaseWebIndexer
+    {
+        public AniLibria(IIndexerConfigurationService configService, Utils.Clients.WebClient wc, Logger l, IProtectionService ps)
+            : base(id: "AniLibria",
+                   name: "AniLibria",
+                   description: "AniLibria is a Public torrent tracker for anime, voiced on russian by AniLibria team",
+                   link: "https://wwww.anilibria.tv/",
```

Comment:
wwww.* ? ;-)

Reply:
Fixed, thank you.

Also added list of things that i need help (consultation) with

Reply:
I'm not a c# dev so I cannot assist with your questions. I've added the team c# dev but he has a lot on his plate ATM so be patient.

Reply:
TY